schema/sale_artwork.js

```diff
@@ -215,16 +218,27 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLInt,
         deprecationReason: 'Favor `counts.bidder_positions`',
       },
-      bid_increment: {
-        type: GraphQLInt,
+      bid_increments: {
+        type: new GraphQLList(GraphQLInt),
         resolve: ({ minimum_next_bid_cents }) => {
           return gravity('/increments').then((res) => {
             const deflt = find(res, { key: 'default' });
-            const bucket = find(deflt.increments, (inc) =>
-              minimum_next_bid_cents >= inc.from &&
-              minimum_next_bid_cents <= inc.to
-            );
-            return bucket.amount;
+            const increments = [minimum_next_bid_cents];
```

Comment:
This got rather hairy to implement—I might be missing a more novel solution. Basically this mutates and array 100 times keeping track of the last bid to predict if the next one is going to land in a new bucket.